examples/benchmarking/run_benchmark.py

```diff
@@ -20,7 +20,25 @@
 
 def main():
     parser = HfArgumentParser(PyTorchBenchmarkArguments)
-    benchmark_args = parser.parse_args_into_dataclasses()[0]
+    try:
```

Comment:
The `run_benchmark.py` script does not accept `--no_` ... arguments anymore. A very explicit error message is given on how the command line args have to be changed. This break backwards compatibility **only** for a command line command and only for the example files, but states explicitly how to change the input name. 
I decided to go for this option because:
a) keeping full backwards compatibility for the example file would have required to do some ugly hacks in the hfArgParser which I don't think is a good idea
b) This only concerns command line commands of example files, so I highly doubt that this change bothers anybody.